handlers/feedback.go

```diff
@@ -163,14 +164,32 @@ func addFeedback(w http.ResponseWriter, req *http.Request, isPositive bool, rend
 		f.URL = "Whole site"
 	}
```

Comment:
I think the block from line 156 to 165 (email and url checks) can be removed, as these checks are performed by feedback API now.

Reply:
Removed.